### ISSUE TITLE
FR: add fake GPS track generator

### ISSUE THREAD

**Issue by afanasy** [Marak/faker.js#221]

**Comment by Marak** [comment]
What is a GPS track? Do you mean generate fake GPS coordinates?

**Comment by afanasy** [comment]
Basically track is just an array of coordinates. But it should be more or less "sane" track, with points not far from each other forming the track, not just random set. I'm working on fleet tracking system and need this for testing.

**Comment by Marak** [comment]
Do you mean a bounding box for longitude / latitude?

Something like this http://janmatuschek.de/LatitudeLongitudeBoundingCoordinates ??

You might be able to google an existing solution.

**Comment by afanasy** [comment]
Thanks!
It's not that I'm unable to find a solution, its just I though it would be a nice additional to the great faker lib.

**Comment by Marak** [comment]
Cool.

Feel free to open up a pull request with the functionality and I'll be glad to merge it.

Thanks!

**Comment by afanasy** [comment]
It's not bounding box. In my app I have a bus following specific track, and I need to emulate this bus for testing - so I need to make a fake track, then to make a fake bus following this track.

**Comment by afanasy** [comment]
Ok, sounds like a plan :)

**Comment by diegolaciar** [comment]
@afanasy  did you find a solution ? I need exactly what you requested on this issue.

"...an array of coordinates. But it should be more or less "sane" track, with points not far from each other forming the track, not just random set..."

Cheers

**Comment by afanasy** [comment]
@diegolaciar not really, I haven't implemented this yet. It should be relatively easy to code geodesic path by incrementing latitude and longitude by some small delta.
